src/cargo/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![deny(unused)]
-#![feature(collections, hash, io, libc, os, path, std_misc, unicode, env, core)]
-#![cfg_attr(test, deny(warnings))]
+#![feature(collections, hash, libc, os, std_misc, unicode, env, core)]
+#![feature(old_io, old_path, str_words)]
+//#![cfg_attr(test, deny(warnings))]
```

Comment:
And just tried with alpha.2 which also fails because the unstable features create a warning making the test build fail.